### PR TITLE
Misc updates 030421a

### PR DESCRIPTION
**Pull request by jbranson000**
Miscellaneous field updates, and inclusion of the Reservation Search mixin.

- Add's new reservation-search mixin for Industry-verticals Travel and Hospitality Industry mixin
- Add's more enum values to room type  property in lodging reservation mixin for Industry-verticals Travel and Hospitality Industry mixin
- Add's new Confirmation Number property in reservation details mixin for Industry-verticals Travel and Hospitality Industry mixin
- Add's Upgrade steps property to upgrade details mixin for Industry-verticals Travel and Hospitality Industry mixin
- Add's Upsell details property to upsell details mixin for for Industry-verticals Travel and Hospitality Industry mixin

### CODE REVIEW

File: components/mixins/experience-event/industry-verticals/experienceevent-upgrade-details.schema.json [hunk 1]
```diff
@@ -31,6 +31,11 @@
         "xdm:upgrade": {
           "type": "object",
           "properties": {
+            "xdm:upgradeSteps": {
+              "title": "Upgrade Steps",
+              "$ref": "https://ns.adobe.com/xdm/datatypes/transaction",
```
Comment (mjasrotia) [inline components/mixins/experience-event/industry-verticals/experienceevent-upgrade-details.schema.json:36]: Should this be "tool-usage" ? majority of step properties are defined as "tool-usage"

File: components/mixins/experience-event/industry-verticals/experienceevent-upsell-details.schema.json [hunk 2]
```diff
@@ -30,6 +30,11 @@
         "xdm:upsell": {
           "type": "object",
           "properties": {
+            "xdm:upsellSteps": {
+              "title": "Upsell Steps",
+              "$ref": "https://ns.adobe.com/xdm/datatypes/transaction",
```
Comment (mjasrotia) [inline components/mixins/experience-event/industry-verticals/experienceevent-upsell-details.schema.json:35]: Should this be "tool-usage" ? majority of step properties are defined as "tool-usage"

Comment (jbranson000) [reply components/mixins/experience-event/industry-verticals/experienceevent-upsell-details.schema.json]: You are right.  I just pushed a fix.

File: components/mixins/experience-event/industry-verticals/experienceevent-upgrade-details.schema.json (hunk 1 above)
Comment (jbranson000) [reply components/mixins/experience-event/industry-verticals/experienceevent-upgrade-details.schema.json]: You are right.  I just pushed a fix.

**Review by mjasrotia — APPROVED** [review]

**Review by prabhum2 — APPROVED** [review]

**Pull request merged**